builtin/game/item.lua

```diff
@@ -284,7 +286,8 @@ function core.item_place_node(itemstack, placer, pointed_thing, param2)
 		place_to = {x = under.x, y = under.y, z = under.z}
 	end
 
-	if core.is_protected(place_to, playername) and
+	-- Do not check protection for nil placer
+	if placer and core.is_protected(place_to, playername) and
```

Comment:
Please let protection mods decide whether they want to allow `""` or not.

Reply:
Ok, thanks